narayana-integration-bom/pom.xml

```diff
@@ -25,7 +25,7 @@
     <!-- narayana dependencies for tomcat and kie server -->
     <version.org.jboss.narayana.tomcat>5.9.0.Final</version.org.jboss.narayana.tomcat>
     <!-- DBCP connection pooling for Narayana -->
-    <version.org.apache.tomcat.tomcat-dbcp>9.0.13</version.org.apache.tomcat.tomcat-dbcp>
+    <version.org.apache.tomcat.tomcat-dbcp>9.0.21</version.org.apache.tomcat.tomcat-dbcp>
```

Comment:
potentially we should update, test and release also narayana-tomcat integration library, as it depends on tomcat too:
https://github.com/web-servers/narayana-tomcat/blob/master/pom.xml#L38